Explorer/Assets/Scripts/CrdtEcsBridge/Components/Transform/SDKTransformSerializer.cs

```diff
@@ -0,0 +1,19 @@
+using CRDT;
+using CrdtEcsBridge.Serialization;
+using System;
+using Utility;
+
+namespace CrdtEcsBridge.Components.Transform
+{
+    public class SDKTransformSerializer : IComponentSerializer<SDKTransform>
+    {
+        public void DeserializeInto(SDKTransform instance, in ReadOnlySpan<byte> data)
+        {
+            var pointer = data;
+            instance.Position.Set(pointer.Read<float>(), pointer.Read<float>(), pointer.Read<float>());
+            instance.Rotation.Set(pointer.Read<float>(), pointer.Read<float>(), pointer.Read<float>(), pointer.Read<float>());
```

Comment:
If the memory is read from data for reach individual byte, is it not possible to copy the entire structure at once? Equivalent of a C++ memcpy from data into instance

Reply:
Yes, you are absolutely right. `Vector3` and `Quaternion` are blittable and have the expected structure.